dahu/core/app/scripts/views/workspace/objects/tooltip.js

```diff
@@ -34,6 +35,10 @@ define([
 
         className: 'object tooltip ui-widget-content',
 
+        initialize: function() {
+            this.changeBackgroundColor();
```

Comment:
My bad, it was in the css and now I put it in the model.
In my next commit I removed the default color from the css file.